src/pages/history.jsx

```diff
@@ -0,0 +1,53 @@
+import { Navbar } from "../components/navbar";
+import { Sidebar } from "../components/sidebar";
+import "../styles/homepage.css";
+import "../root.css";
+import { useState } from "react";
+const History = () =>{
+    const [sidebar, setSidebar] = useState(true);
+    const [ drop, setDrop ] = useState(false);
+    return (
+        <div className="App">
+            <Navbar sidebar={sidebar} setSidebar={setSidebar}/>
+            <main className="main-cont">
+                {sidebar ? <Sidebar/> : null}
```

Comment:
you can use  && operator here.

Reply:
Ohh that is used for this purpose? Will change it!